packages/ilib-loctool-jst/package.json

```diff
@@ -0,0 +1,64 @@
+{
+    "name": "ilib-loctool-jst",
+    "version": "1.1.3",
+    "main": "./JavaScriptFileType.js",
+    "description": "A loctool plugin that knows how to process Javascript Template (JST) files",
+    "license": "Apache-2.0",
+    "keywords": [
+        "internationalization",
+        "i18n",
+        "localization",
+        "l10n",
+        "globalization",
+        "g11n",
+        "strings",
+        "resources",
+        "locale",
+        "translation",
+        "html",
+        "javascript",
+        "template"
+    ],
+    "email": "ehoogerbeets@gmail.com",
+    "author": {
+        "name": "Edwin Hoogerbeets",
+        "web": "http://www.translationcircle.com/",
+        "email": "ehoogerbeets@gmail.com"
+    },
+    "contributors": [
+        {
+            "name": "Edwin Hoogerbeets",
+            "email": "ehoogerbeets@gmail.com"
+        }
+    ],
+    "files": [
+        "README.md",
+        "LICENSE",
+        "HTMLTemplateFileType.js",
+        "HTMLTemplateFile.js"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/iLib-js/ilib-mono/packages/ilib-loctool-jst.git"
+    },
+    "scripts": {
+        "dist": "npm pack",
+        "test": "pnpm jest",
+        "test:watch": "pnpm jest --watch",
+        "debug": "node --inspect-brk node_modules/.bin/jest -i",
```

Comment:
Stuff from https://github.com/iLib-js/ilib-mono/pull/14 applies to this PR as well